app/views/how-tos/index.html

```diff
@@ -14,13 +14,33 @@
 {% block content %}
   <div class="nhsuk-grid-row">
 
-    <div class="nhsuk-grid-column-two-thirds">
+    <div class="nhsuk-grid-column-full">
 
       <h1>Guides</h1>
 
       <p class="nhsuk-lede-text">These guides will show you how to use the prototype kit, from creating a simple page to building complex user journeys.</p>
 
-      <h2>Build a basic prototype</h2>
+    </div>
+
+
+
+    <div class="nhsuk-grid-column-one-half">
```

Comment:
Needs an extra “row” div here I think? (Might not be strictly required but I’ve always done it wherever a new “row” starts)

```suggestion
  </div>
  <div class=“nhsuk-grid-row”>
    <div class="nhsuk-grid-column-one-half">
```

Reply:
lol doing this completely broke the formatting 🙃 

![page with extra divs and broken formatting](https://github.com/user-attachments/assets/4c1cb4df-960e-40f1-a454-1c9c4b1d0247)

so rolling back

Reply:
Actually no this was me - got the full width column wrong :D